### DISCUSSION TITLE
PhotoSync favorites for already sync’d photos

### DISCUSSION THREAD

**Discussion by egut125** [photoprism/photoprism#2128] [Q&A] [answered]
If I've already transferred a photo to photoprism with PhotoSync I cannot favorite that photo on iOS and then re sync it with PhotoSync to photoprism and get the favorite to be set. I think it makes the yaml file but it doesn't show as a favorite in the UI. 

If I already have the photo as a favorite the first time PhotoSync syncs it to photo prism then it shows correctly in the UI. 

Anyone else experience anything similar?

Thanks!

**Comment by lastzero** [comment, marked as answer]
Works as designed. It should not overwrite flags of existing pictures. Same for direct, manual changes in YAML files.

**Reply by egut125** [reply]
Hmm gotcha. Thanks for the response! So no way to edit those flag outside of the UI then? No command line, api, or merging of yaml type things?

I really like the photosync sending favorites to photoprism but sometimes I favorite them  after PhotoSync has ran once. Was extremely convenient having photoprism be the large back up photos and my favorite can flow seamlessly from my phone. But missing some favorites if I'm slow isn't ideal.
